examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<blockquote>To run our algorithm for a long time, we can run it asynchronously with Amazon Braket Hybrid Jobs</blockquote><p>You can execute long-running algorithm asynchronously with Amazon Braket Hybrid Jobs, which...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>